client/my-sites/people/team-list/index.jsx

```diff
@@ -3,33 +3,66 @@
  */
 import { localize } from 'i18n-calypso';
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { errorNotice, removeNotice } from 'calypso/state/notices/actions';
 
 /**
  * Internal dependencies
  */
-import SiteUsersFetcher from 'calypso/components/site-users-fetcher';
 import Team from './team';
+import useUsers from 'calypso/data/users/use-users';
 
-class TeamList extends React.Component {
-	static displayName = 'TeamList';
+function TeamList( props ) {
```

Comment:
Nit: I wonder if we can make this `props` object more explicit here, and specify what props we're actually expecting. Over time I've found components that receive unknown props and pass them down to create a lot of noise and can be sometimes problematic.

Reply:
Agreed. Added here: https://github.com/Automattic/wp-calypso/pull/50712/commits/78b58bdf4b44bd779b1b3fea45326de58619e5e6